Resources/config/core.xml

```diff
@@ -34,11 +34,21 @@
         </service>
 
         <!-- Services used to format the label, default is sonata.admin.label.strategy.noop -->
-        <service id="sonata.admin.label.strategy.bc" class="Sonata\AdminBundle\Translator\BCLabelTranslatorStrategy" />
-        <service id="sonata.admin.label.strategy.native" class="Sonata\AdminBundle\Translator\NativeLabelTranslatorStrategy" />
-        <service id="sonata.admin.label.strategy.noop" class="Sonata\AdminBundle\Translator\NoopLabelTranslatorStrategy" />
-        <service id="sonata.admin.label.strategy.underscore" class="Sonata\AdminBundle\Translator\UnderscoreLabelTranslatorStrategy" />
-        <service id="sonata.admin.label.strategy.form_component" class="Sonata\AdminBundle\Translator\FormLabelTranslatorStrategy" />
+        <service id="sonata.admin.label.strategy.bc"
+            class="Sonata\AdminBundle\Translator\BCLabelTranslatorStrategy"
+            public="false" />
```

Comment:
Theses services are supposed to be injected into the admin service. Their purpose is to help generating a translation string from a form field name. They are not supposed to be retrieved from the container in the user's code. Having them private still allows to inject them, but does not expose them.